manifests/n/Nvidia/App/11.0.1.189/Nvidia.App.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Nvidia.App
+PackageVersion: 11.0.1.189
+Platform:
+  - Windows.Desktop
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: -s
+  SilentWithProgress: -passive
+  Custom: -noreboot
+InstallerSuccessCodes:
+- 3858759680
```

Comment:
This is not an installer Success Code. This is an error code indicating that no compatible GPU was found
![image](https://github.com/user-attachments/assets/dafa69b8-80d4-4415-b119-8751de292084)